### ISSUE TITLE
Latest version installed but old version running.

### ISSUE THREAD

**Issue by mojosam** [ytdl-org/youtube-dl#21803]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2019.07.16**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
youtube-dl -v -f bestvideo[ext=mp4]+bestaudio[ext=m4a]/mp4 --verbose https://www.youtube.com/watch?v=dcxapaTMnGw
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-f', 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/mp4', '--verbose', 'https://www.youtube.com/watch?v=dcxapaTMnGw']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.04.07
[debug] Python version 3.7.3 (CPython) - Darwin-17.7.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 4.1.3, ffprobe 4.1.3, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] dcxapaTMnGw: Downloading webpage
[youtube] dcxapaTMnGw: Downloading video info webpage
ERROR: dcxapaTMnGw: "token" parameter not in video info for unknown reason; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/youtube_dl/YoutubeDL.py", line 796, in extract_info
    ie_result = ie.extract(url)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/youtube_dl/extractor/common.py", line 529, in extract
    ie_result = self._real_extract(url)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/youtube_dl/extractor/youtube.py", line 1693, in _real_extract
    video_id=video_id)
youtube_dl.utils.ExtractorError: dcxapaTMnGw: "token" parameter not in video info for unknown reason; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```

## Description

I'm trying to download a YouTube video. I'm getting the error above. However, I noticed that the version listed above (2019.04.07) is not the version of youtube-dl that I have installed.

I'm on a Mac. When I type:
brew upgrade youtube-dl

Homebrew reports:
Error: youtube-dl 2019.07.14 already installed

I don't know if this is a Homebrew problem or youtube-dl problem.

**Comment by Hrxn** [comment]
Try `which youtube-dl`

**Comment by mojosam** [comment]
Aha! So my system is messed up. I had previously attempted to install youtube-dl via pip, but that didn't seem to work. I uninstalled it (or maybe I just deleted what I saw) then installed Homebrew and then youtube-dl via Homebrew.

So when I type:
which youtube-dl

It responds with:
/Library/Frameworks/Python.framework/Versions/3.7/bin/youtube-dl

The date of the version in there is 4/11/19. That's obviously what's running.

So what is the cleanest way for me to get rid of that version and convince my computer to look in /usr/local/bin instead?
